lisp/plugins/list_layout/layout.py

```diff
@@ -257,7 +266,10 @@ def _key_pressed(self, event):
             sequence = keyEventKeySequence(event)
             if sequence in self._go_key_sequence:
                 event.accept()
-                self.__go_slot()
+                if self._running_model.is_playing() is not True:
+                    self.__go_slot()
+                if self._get_go_key_disabled_while_playing() is not True:
+                    self.__go_slot()
```

Comment:
I think it should be something like this:
```python
if not self.go_key_disabled_while_playing or not self._running_model.is_playing():
    self.__go_slot()
```

Reply:
I was debating this, thought this looked cleaner in terms of line length, but I forgot about returning explicitly so the next case is not checked. 

I'll get this tweaked and test out my changes given the notes you've made above